ui/app/AppLayouts/Chat/controls/community/CommunityOptions.qml

```diff
@@ -9,8 +9,6 @@ import StatusQ.Controls 0.1
 ColumnLayout {
     id: root
 
-    property alias archiveSupportOptionVisible: archiveSupport.visible
```

Comment:
Shouldn't the whole `archiveSupport` down below be removed? It's just dead code imho

Reply:
It's a bit confusing, but no.

The things below are **per community**. The setting we removed was just the "global" setting. Now, we see the per community setting by default.

I also changed the default to `checked`, but we want to let users turn it off if needed.

I hope I explained it correctly